mon-pix/tests/helpers/setup-integration.js

```diff
@@ -0,0 +1,7 @@
+import setupIntl from './setup-intl';
+import { setupRenderingTest } from 'ember-mocha';
+
+export default function setupIntegration() {
```

Comment:
Est-ce qu'on ne devrait pas deplacer ce code dans tests/test-helper.js et ainsi ne pas modifier les fichiers de tests?

Reply:
j’ai tenté de le mettre dans le test-helper.js, mais il a besoin du this.owner  pour avoir le service, qui est présent dans les tests d’intégration. Ca me fait donc des erreurs sur les autres tests. Ca me parait compliqué